src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -118,17 +118,34 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="InitialJsonParseErrorText" xml:space="preserve">
-    <value>Settings could not be loaded from file - temporarily using the default settings. Check for syntax errors, including trailing commas.</value>
+    <value>Settings could not be loaded from file - temporarily using the default settings. Check for syntax errors, including trailing commas.
+</value>
+  </data>
+  <data name="MissingDefaultProfileText" xml:space="preserve">
+    <value>Could not find your default profile in your list of profiles - temporarily using the first profile. Check to make sure the defaultProfile matches the guid of one of your profiles.
+</value>
+  </data>
+  <data name="DuplicateProfileText" xml:space="preserve">
+    <value>Found multiple profiles with the same guid in your profiles - temporarily ignoring duplicates. Make sure each profile's guid is unique.
+</value>
+  </data>
+  <data name="NoProfilesText" xml:space="preserve">
+    <value>Failed to load settings - no profiles were found in your settings.

```

Comment:
"Failed to load settings" is implied by the earlier two lines in the dialog ("We failed to load your settings.")